components/org.wso2.carbon.identity.application.authenticator.oidc/src/main/java/org/wso2/carbon/identity/application/authenticator/oidc/OpenIDConnectAuthenticator.java

```diff
@@ -235,22 +235,15 @@ protected String getState(String state, Map<String, String> authenticatorPropert
     /**
      * @return
      */
-    protected String getScope(String scope, Map<String, String> authenticatorProperties) {
+    protected String getScope(Map<String, String> authenticatorProperties) {
```

Comment:
we can't change the existing method signatures. There could be some authenticators extended from this, may use this method.

. Let's overload this method as getScope(Map<String, String> authenticatorProperties), and return the scope from the authenticator properties